src/utils/classNameUseCheckInjector.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import ReactDOM from 'react-dom';
+import getComponentName from './getComponentName';
+
+export default (target: Object) => {
+  let elementClassName = '';
+
+  const targetCDM = target.componentDidMount;
+
+  // eslint-disable-next-line no-param-reassign
+  target.componentDidMount = function componentDidMount() {
+    if (typeof targetCDM === 'function') {
+      targetCDM.call(this);
+    }
+
+    const classNames = elementClassName.split(' ');
+    // eslint-disable-next-line react/no-find-dom-node
+    const node: Element | null = (ReactDOM.findDOMNode(this): any);
```

Comment:
Running React in Strictmode gives this warning:
```
Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of StyledComponent which is inside StrictMode. Instead, add a ref directly to the element you want to reference.
```

I am aware that it is only run in development-mode, but it clutters console output when trying to find errors.

@Fer0x Could we change this here somehow? (by not using findDOMNode)